cmd/coronabot/main.go

```diff
@@ -22,28 +26,45 @@ func main() {
 	log.Println("Starting CoronaBot")
 
 	slackToken := os.Getenv("SLACK_TOKEN")
-	if slackToken == "" {
-		log.Print("no slack token provided. Set SLACK_TOKEN environment variable")
-		os.Exit(1)
-	}
+	telegramToken := os.Getenv("TELEGRAM_TOKEN")
+
+	var messageSender model.MessageSender
```

Comment:
This block is getting a little too big, would you mind creating a `MessageSenderFactory` to abstract the creation of these objects?